src/VoiceConnection.ts

```diff
@@ -58,6 +61,26 @@ export type VoiceConnectionState =
 			status: VoiceConnectionStatus.Destroyed;
 	  };
 
+/**
+ *
+ */
+export enum VoiceConnectionEvents {
+	StateChange = 'stateChange',
+	JoinVoiceChannel = 'joinVoiceChannel',
+	Error = 'error',
+	Debug = 'debug',
+}
+
+export interface SignalJoinVoiceChannelPayload {
+	op: GatewayOPCodes.VoiceStateUpdate;
+	d: {
+		guild_id: string;
```

Comment:
```suggestion
		guild_id: `${bigint}`;
```

Reply:
Resolved in https://github.com/discordjs/voice/pull/65/commits/54f4d50d344cb7353147c09e1b26eef7bdcc28fd